NuRadioReco/modules/channelCWNotchFilter.py

```diff
@@ -1,3 +1,6 @@
+import logging
+logger = logging.getLogger(__name__)
```

Comment:
can you please check the PR about logging (which gets merged to day) and initialize the logger according to the guidelines introduced there?